### PR TITLE
migrate controller poolcoordinatorcert 

### PR DESCRIPTION
**Pull request by luc99hen**
#### What type of PR is this?
/kind enhancement

#### What this PR does / why we need it:
1. migrate poolcoordinator-cert-manager into yurt-manager framework
2. fix syntax in helm charts 

#### Which issue(s) this PR fixes:

Fixes # https://github.com/openyurtio/openyurt/issues/1067

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by rambohe-ch** [comment]
@luc99hen Maybe you can use `injector` to prepare `kubernetes.Interface`, and you reference the `injector` in this pr: https://github.com/openyurtio/openyurt/pull/1334

File: pkg/controller/poolcoordinator/cert/poolcoordinatorcert_controller.go [hunk 1]
```diff
@@ -175,84 +187,76 @@ var allSelfSignedCerts []CertConfig = []CertConfig{
...
-		DeleteFunc: func(obj interface{}) {},
-	})
+var _ reconcile.Reconciler = &ReconcilePoolcoordinatorcert{}
 
-	return &certManager
+// ReconcilePoolcoordinatorcert reconciles a Poolcoordinatorcert object
+type ReconcilePoolcoordinatorcert struct {
+	Client   client.Interface
+	scheme   *runtime.Scheme
+	recorder record.EventRecorder
```
Comment (rambohe-ch) [inline pkg/controller/poolcoordinator/cert/poolcoordinatorcert_controller.go:207]: It seems that `scheme` and `recorder` are unused

File: pkg/controller/poolcoordinator/cert/poolcoordinatorcert_controller.go [hunk 2]
```diff
@@ -175,84 +187,76 @@ var allSelfSignedCerts []CertConfig = []CertConfig{
 	},
 }
 
-// PoolCoordinatorCertManager manages certificates releted with poolcoordinator pod
-type PoolCoordinatorCertManager struct {
-	kubeclientset client.Interface
-	podLister     corelisters.PodLister
-	podSynced     cache.InformerSynced
```
[the remark this answers is not in the document]
Comment (luc99hen) [reply pkg/controller/poolcoordinator/cert/poolcoordinatorcert_controller.go:182]: Currently, we don't need podLister and podSynced for `Reconcile()`.

File: pkg/controller/poolcoordinator/cert/poolcoordinatorcert_controller.go (hunk 1 above)
Comment (luc99hen) [reply pkg/controller/poolcoordinator/cert/poolcoordinatorcert_controller.go]: done

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**